plat/common/arm/traps_arm64.c

```diff
@@ -208,5 +208,5 @@ void trap_el1_irq(void)
 		UK_CRASH("EL1 async tag check fault\n");
 #endif /* CONFIG_ARM64_FEAT_MTE */
 
-	gic->ops.handle_irq();
+	gic->ops.handle_irq(regs);
```

Comment:
I believe you wanted to pass `regs` as an argument to `trap_el1_irq` as well?

Reply:
+1. This will also require changes to the ARM assembly for trap handling just like with x86.